server.js

```diff
@@ -28,8 +114,73 @@ app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
+
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('hello world')
+})
+
+//this shows all nominees
+app.get('/nominees', async (req, res) => {
+  const nominees = await Nominee.find()
+    res.json(nominees)
+})
+
+//this shows all films selected year  
+app.get('/year/:year', (req, res) => {
+  Nominee.find({'year_film': req.params.year})
+    .then((results) => {
+    console.log("hej")
+    res.json(results);
+  }).catch((err) => {
+    console.log("hej2")
+    res.json({message: 'can not find year', err: err});
+  });
+})
+
+//this shows avatar written in any way
+app.get('/avatar', (req, res) => {
+  Nominee.find({'nominee': /Avatar/i})
+    .then((results) => {
+    console.log("hej")
+    res.json(results);
+  }).catch((err) => {
+    console.log("hej2")
+    res.json({message: 'can not find nominee', err: err});
+  });
+})
+
+//this shows any result wich includes searchstring (for exampel: nomineesQuery?q=swe, which returns the film: "One Chance" since it has the letters swe in the nominee)
+app.get('/nomineesQuery', (req, res) => {
```

Comment:
Including this logic in the `/nominees` endpoint could work as well (more RESTful)